.github/workflows/export.yml

```diff
@@ -13,10 +14,11 @@ concurrency:
 
 env:
   GODOT_VERSION: 4.4.1
+  PCK_NAME: threadbare
 
 jobs:
   build:
-    name: Build for web
```

Comment:
As you have noticed, this job is "api" in that it's used in the repo's "required status checks" setting. You could change that setting, or more likely, disable it temporarily and update it once the branch is merged.

![image](https://github.com/user-attachments/assets/e472923d-37ee-4301-8d4b-5a77c6b1987f)